src/main/data_structure/list_exercise.py

```diff
@@ -77,3 +76,11 @@ def sort_list_bubble_dsc(list_of_number):
     return list_of_number
 
 
+def even_finder(list_of_number):
```

Comment:
function names should be verb
for this example we can use **find_even_numbers** instead of **even_finder**
@Ahad726